lib/eslint-patch.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+var meld = require('meld');
+var docs = require('./docs');
+var Config = require("eslint/lib/config");
+var ConfigUpgrader = require('./config_upgrader');
+
+var supportedPlugins = ['react', 'babel'];
+
+module.exports = function patcher(eslint) {
+
+  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
Looking at the history it seems to be added while upgrading to 3, that's why I am confuse.
Anyways I am digging into eslint to understand where did this logic go.